src/Peering/Peering/Az.Peering.psd1

```diff
@@ -121,7 +121,7 @@ PrivateData = @{
         # IconUri = ''
 
         # ReleaseNotes of this module
-        ReleaseNotes = '* Preview Release for Microsoft Peering Service'
+        ReleaseNotes = '* Updated AutoMapper to 6.2.2'
```

Comment:
@rileymckenna you can revert the change made here -- we'll do this as a part of versioning